### ISSUE TITLE
Training accuracy on test set is not improving (tested on two different datasets)

### ISSUE THREAD

**Issue by hcmea** [facebookresearch/ConvNeXt#39]
Thanks for sharing this code. 

I tried to run training script on my custom dataset (~50k samples) and somehow I could not get improvement on classification accuracy even after 300 epochs.  

!python main.py --model convnext_tiny --drop_path 0.1  --data-path "/my/custom/dataset"  --data_set "customDS" --nb_classes 2 --batch_size 256 --lr 4e-2 --update_freq 4 --model_ema true --model_ema_eval true 

Epoch: [0] Total time: 0:00:41 (1.1597 s / it)
Averaged stats: lr: 0.003596  min_lr: 0.003596  loss: 2.2043 (2.8601)  class_acc: 0.5234 (0.5078)  weight_decay: 0.0500 (0.0500)
Accuracy of the model on the 2297 test images: 44.4%
Max accuracy: 44.36%

Epoch: [299] Total time: 0:00:22 (0.6137 s / it)
Averaged stats: lr: 0.000010  min_lr: 0.000010  loss: 0.6883 (0.6902)  class_acc: 0.5352 (0.5386)  weight_decay: 0.0500 (0.0500)
Accuracy of the model on the 2297 test images: 55.6%
Max accuracy: 55.86%

I trained the same custom dataset using ResNet101 and it achieved 90% classification accuracy on the same test set.

I also repeat training on Imagenette dataset (a subset of ImageNet dataset with 10 easy classes) and the same thing happened. No substantial decrease in training loss and almost no improvement on the classification accuracy. 
https://github.com/fastai/imagenette

!python -m torch.distributed.launch --nproc_per_node=2 main.py --model convnext_tiny --drop_path 0.1 --batch_size 64 --lr 4e-3 --update_freq 4 --model_ema true --model_ema_eval true --data_path  "/imagenette/with/10/classes" --data_set "IMNET" --output_dir "/my/output/path" --nb_classes 10

Epoch: [284] Total time: 0:00:34 (0.3696 s / it)
Averaged stats: lr: 0.000029  min_lr: 0.000029  loss: 2.0305 (2.0208)  weight_decay: 0.0500 (0.0500)
Accuracy of the model on the 1100 test images: 1.2%
Max accuracy: 9.91%

How can I improve the training performance? Did you test the training parameters on smaller size datasets?

Thanks!

**Comment by developer0hye** [comment]
@hcmea 

Did you train ResNet101 model with same code with this project on your custom dataset?

**Comment by liuzhuang13** [comment]
Hi,

Thanks @developer0hye, good point indeed. @hcmea did you use the same training code for ResNet and ConvNeXt? If not I suspect this is due to training settings, not model architectures. I suggest trying training ConvNeXt models using the same training code as ResNet in that case.

**Comment by hcmea** [comment]
I trained ResNet101 with it's own training settings. I will try training ConvNext with ResNet training parameters and let you know when I got the results. It is probably the training parameters need to be tuned for smaller size datasets.

**Comment by vaibhavpawar05** [comment]
Hi @hcmea, I am curious if you were able to resolve this? I want to do a similar experiment with my custom dataset (100k images) and suspect that the result from running the ConvNeXt scripts with default parameters might not be good. It would be great if you could share what worked for you in case you have resolved this. Thank you.

**Comment by hcmea** [comment]
@vaibhavpawar05, I made some changes in main.py to train the original ConvNext model on my customized data. I did the following changes and it worked:
sampler_train = None
sampler_val = None 
mixup_active = None

I disabled torch.utils.data.DistributedSampler and use my own augmentations specific to my dataset. Then I started to get meaningful results.

**Comment by lxwkobe24** [comment]
> @vaibhavpawar05, I made some changes in main.py to train the original ConvNext model on my customized data. I did the following changes and it worked: sampler_train = None sampler_val = None mixup_active = None
> 
> I disabled torch.utils.data.DistributedSampler and use my own augmentations specific to my dataset. Then I started to get meaningful results.

您好请问，要怎么禁用您说的这几个
